src/slides/1-demos/chevres/demo/component/styles.css

```
@@ -0,0 +1,23 @@
+html,
+body {
+  height: 100%;
+  background: white;
+  font-family: "Inter", sans-serif;
+}
+
+body {
+  margin: 0;
+  display: grid;
+  place-items: center;
+  text-wrap: pretty;
+  background: rgb(230, 230, 230);
+}
+
+article {
+  min-width: 200px;
+  max-width: min(90vw, 1000px);
+  resize: horizontal;
+  overflow: hidden;
+  background: white;
+  border-radius: 0.75rem;
```

un peu de padding pour aérer, et augmentation du radius car radius du container = radius du contenu (image) + padding
```suggestion
  padding: 1em;
  border-radius: 1.75rem;
```